### DISCUSSION TITLE
Cfclient installation error

### DISCUSSION THREAD

**Discussion by anoopk18** [bitcraze/discussions#526] [Q&A]
Hi, 

I have been trying to get started with the [cfclient installation](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/installation/install/), but I have been running into this error on ubuntu. Could you please advise how to troubleshoot this error? 

Git is already installed and I can also see that it is an updated version.

amath@DESKTOP-CPPVCSE:~/sample-directory$ git clone https://github.com/bitcraze/crazyflie-lib-python.git
Cloning into 'crazyflie-lib-python'...
fatal: unable to access 'https://github.com/bitcraze/crazyflie-lib-python.git/': Failed to connect to 192.91.225.210 port 44 after 130542 ms: Connection timed out

**Comment by knmcguire** [comment]
Hi! This seems to be a git issue and not a client issue.... Either there is something going on with their services or something going on with your firewall, but that is something that you would need to figure out.

I see that you are following the 'install from source' instructions. Are you doing any development for the cfclient? If not you can just install it with pip.

```
pip3 install cfclient
```
